config.py

```diff
@@ -37,7 +40,7 @@ def load_config(config_file):
             raise Exception("Your config.yml has the default Lichess API token. This is probably wrong.")
 
         if not os.path.isdir(CONFIG["engine"]["dir"]):
-            raise Exception("Your engine directory `{}` is not a directory.".format(CONFIG["engine"]["dir"]))
```

Comment:
I think it's better to keep this line that remove it.

Reply:
It changed in ShailChoksi's repo. why they changed

Reply:
No they didn’t change that in ShailChoksi’s repo, it’s an error fix.

Reply:
@TheYoBots
I didn't understand. can you fix this in this branch

Reply:
Yes I will as soon as I can.